pymc_marketing/mmm/tvp.py

```diff
@@ -236,4 +238,4 @@ def infer_time_index(
 
     Infers the time-indices by calculating the number of days since the first date in the dataset.
     """
-    return (date_series_new - date_series[0]).dt.days.values // time_resolution
+    return (date_series_new - date_series[0]).values // time_resolution
```

Comment:
I am noticing this when date_col is not datetime64

Does this support that case still?

Reply:
Done. Changed to actual code